paperqa/agents/search.py

```diff
@@ -400,7 +400,22 @@ async def get_saved_object(
         return None
 
     def clean_query(self, query: str) -> str:
-        for replace in ("*", "[", "]", ":", "(", ")", "{", "}", "~", '"'):
+        for replace in (
+            "*",
+            "[",
+            "]",
+            ":",
+            "(",
+            ")",
+            "{",
+            "}",
+            "~",
+            '"',
+            "^",
+            ">",
+            "<",
+            "+",
```

Comment:
Can we do `for replace in "*[]:(){}!"` (adding the other chars too)

It works the same and is less code

Reply:
Yes but will do a separate one for 
`query = query.replace('"', "")`